Makefile

```diff
@@ -1,40 +1,39 @@
 ATLAS_URIS = "$(ATLAS_FREE)" "$(ATLAS_REPLSET)" "$(ATLAS_SHARD)" "$(ATLAS_TLS11)" "$(ATLAS_TLS12)" "$(ATLAS_FREE_SRV)" "$(ATLAS_REPLSET_SRV)" "$(ATLAS_SHARD_SRV)" "$(ATLAS_TLS11_SRV)" "$(ATLAS_TLS12_SRV)" "$(ATLAS_SERVERLESS)" "$(ATLAS_SERVERLESS_SRV)"
-GODISTS=linux/amd64 linux/386 linux/arm64 linux/arm linux/s390x
 TEST_TIMEOUT = 1800
 
 ### Utility targets. ###
 .PHONY: default
-default: add-license build build-examples check-env check-fmt check-modules lint test-short
+default: add-license build build-tests check-fmt check-modules lint test-short
 
+# Find all .go files not in the vendor directory and try to write a license notice. Then check for
+# any changes made with -G. to ignore permissions changes. Exit with a non-zero exit code if there
+# is a diff.
 .PHONY: add-license
 add-license:
-	# Find all .go files not in the vendor directory and try to write a license notice.
 	find . -path ./vendor -prune -o -type f -name "*.go" -print | xargs ./etc/add_license.sh
-	# Check for any changes made with -G. to ignore permissions changes. Exit with a non-zero
-	# exit code if there is a diff.
 	git diff -G. --quiet
 
 .PHONY: build
 build:
 	go build $(BUILD_TAGS) ./...
```

Comment:
How do you like to add `build-tests` as a dependency of the `build`?

Reply:
I tried that, but ran into 2 issues:
1. The test code relies on Go 1.19 or newer libraries, so we can't build them in the Go 1.13 build task.
2. The `build-tests` target actually runs the test binaries (but skips all the tests), so the cross-compile tasks (i.e. 386, arm, arm64, s390x) won't work because the compiled binary is incompatible with the host's CPU architecture.

We can resolve those issues:
1. Run `make build -o build-tests` to omit the `build-tests` dependency for the Go 1.13 build.
2. Either don't include `build-tests` as a dependency of `build-no-tags` (which is used by the cross-compile tasks), or just remove the `build-no-tags` Makefile target and run `go build ./...` directly for the cross-compile tasks.

I'll try to make those changes.